root/admin/attributes/form.tt

```diff
@@ -7,7 +7,11 @@
         [% form_row_text(r, 'iso_code_2b', add_colon(l('ISO 639-2/B'))) %]
         [% form_row_text(r, 'iso_code_2t', add_colon(l('ISO 639-2/T'))) %]
         [% form_row_text(r, 'iso_code_3', add_colon(l('ISO 639-3'))) %]
-        [% form_row_text(r, 'frequency', add_colon(l('Frequency'))) %]
+        [% WRAPPER form_row %]
+            [% r.label('frequency', add_colon(l('Frequency'))) %]
+            [% r.number('frequency') %]
+            [% field_errors(form, 'frequency') %]
+        [% END %]
```

Comment:
It's an int, so this seems preferable? (also not sure why we don't have form_row_number or form_row_int as a macro, it seems to be used in quite a few places)

Reply:
Seems unrelated to MBS-10100, and should probably be wrapped in `form_row_select` with options labelled after https://github.com/metabrainz/musicbrainz-server/blob/v-2019-05-13-schema-change/root/admin/attributes/form.tt#L12-L18

Reply:
Split it into its own commit now. Not sure about the select, what's the big problem with having it as an int field just as it was?

Reply:
`select` prevents entering invalid values and replaces integer with human-readable label.

The same reasons we don’t have an int field for language, vote, work type, and so on.

Reply:
Yes, but we auto-generate the select options from the DB for those. I guess we could give this a static 0, 1, 2 set of options, if you want, I can try to figure it out. I still feel the current text description for frequency under the field is better than a shortened select label though, and adding the entire lines as a select would look weird.

Reply:
(the range we added here already should prevent adding invalid values)

Reply:
Not for all of those, vote options are defined [in constants](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-06-03/lib/MusicBrainz/Server/Constants.pm#L304-L307), not from the database.

Labels could be “Frequent”, “Other”, “Hidden”. The rest of the explanation for 0 could be kept apart. Then, the field should probably renamed “Display Group” or something.

Reply:
Oh! That might make sense then. I'll look into it!

Reply:
Changed it to use a select, please review :)